src/Command/Checker.php

```diff
@@ -54,13 +54,15 @@ protected function configure()
                 'exclude-dir',
                 null,
                 InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
-                'Prevent directory to be used when check for doc headers'
+                'Exclude the specified directory from being scanned; declare multiple directories ' .
+                'with multiple invocations of this option.'
```

Comment:
Usually I see concatentation operators on the _next_ line, to make it clear that the string is a continuation of the previous line; otherwise, it looks like you have an extra argument.

Otherwise, :+1: for the new message.